apps/artwork/components/auction/templates/bid.jade

```diff
@@ -12,16 +13,25 @@ if auction.is_open && !artwork.is_sold && !auction.is_live_open
       //- Rendered client-side
     if user
       .artwork-auction__bid-form__max-bid
-        if me && me.lot_standing.is_highest_bidder
-          .artwork-auction__bid-form__max-bid-label.is_winning
-            | You're the highest bidder
-        else if me && !me.lot_standing.is_highest_bidder
-          if sale_artwork.reserve_message === 'Reserve not met'
-            .artwork-auction__bid-form__max-bid-label.is_losing
-              | Reserve not met, increase your bid
+        if me && me.lot_standing
+          if leadingBidder && !reserveNotMet
+            .artwork-auction__bid-form__max-bid-label.is-winning
+              | Highest Bidder
+              i.icon-bid-arrow.up
+                  include ../../../../../components/main_layout/public/icons/bid-status-arrow.svg
           else
-            .artwork-auction__bid-form__max-bid-label.is_losing
-              | You've been outbid, increase your bid
+            if leadingBidder && reserveNotMet
```

Comment:
It seems there's a chance to extract this bit of templating/styling into its own component. For instance I noticed this line `if leadingBidder && reserveNotMet` is found in...

`apps/artwork/components/auction/templates/bid.jade`
`apps/auction/templates/my_active_bids.jade`
`components/my_active_bids/template.jade`

Not a blocker, but something worth considering following up with a refactor for.